### PR TITLE
Add check for network access and docker installation 

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

**Comment by rafaellehmkuhl** [comment]
Fix #106

File: install/install.sh [hunk 1]
```diff
@@ -41,7 +41,12 @@ NECESSARY_SPACE_GB=4
 
 # Check for docker and install it if not found
 echo "Checking for docker."
-docker --version || curl -fsSL https://get.docker.com | sh && systemctl enable docker
+docker --version || curl -fsSL https://get.docker.com | sh || (
+    echo "Failed to start docker, something is wrong."
+    echo "Please report this problem."
+    exit 1
+)
+systemctl enable docker
```
Comment (rafaellehmkuhl) [inline install/install.sh:49]: I assume you were not able to reproduce #106, right?

**Review by rafaellehmkuhl — APPROVED** [review]

File: install/install.sh (hunk 1 above)
Comment (patrickelectric) [reply install/install.sh:49]: Nops, I got a network problem where my avahi work messed up with devconf and the result was that get.docker was failing. 

The install script is working as intended

File: install/install.sh [hunk 2]
```diff
@@ -17,6 +17,12 @@ set -e
 # Check if the script is running as root
 [[ $EUID != 0 ]] && echo "Script must run as root."  && exit 1
 
+echo "Check if network and remote are available."
+curl -fsSL --silent $REMOTE/install/install.sh &> /dev/null || (
```
Comment (jaxxzer) [inline install/install.sh:21]: I think we should only redirect stdout, stderr would be useful here?

Comment (patrickelectric) [reply install/install.sh]: done

**Pull request merged**